app/src/components/market/profile/closed_market_detail.tsx

```diff
@@ -147,7 +149,17 @@ export const ClosedMarketDetailWrapper = (props: Props) => {
         <SubsectionTitle>Details</SubsectionTitle>
         <Grid>
           <TitleValue title="Category" value="Politics" />
-          <TitleValue title="Arbitrator" value="realit.io and dxDAO" />
+          {arbitrator && (
+            <TitleValue
+              title={'Arbitrator'}
+              value={[
```

Comment:
I would prefer to use a fragment instead of an array. Or, even better, to make a small component (in the same file) to render this.

Reply:
Also I make a refactoring in all the application